src/game/PowerGrid.hxx

```diff
@@ -0,0 +1,72 @@
+#ifndef POWERGRID_HXX_
+#define POWERGRID_HXX_
+
+#include "point.hxx"
+#include <vector>
+
+struct PowerNode
+{
+  Point coordinate;
+  int powerProduction = 0;
+};
+
+class PowerGrid;
+void mergePowerGrids(PowerGrid &mainGrid, PowerGrid &toBeMerged);
+class PowerGrid
+{
+public:
+  PowerGrid(PowerNode powerNode);
+
+  size_t size() { return m_powerNodes.size(); };
+
+  /**
+   * @brief Add a powernode to this powergrid
+   * 
+   * @param zoneNode to add
```

Comment:
might want to change it to "powerNode Power node to add"

Reply:
copy and waste 🙈